### ISSUE TITLE
Undefined array key "traceHash" Error

### ISSUE THREAD

**Issue by khalid-eyez** [bedezign/yii2-audit#294]
I have installed this extension the lastest version 1.2.7 following the documentation and works well except when clicking the database panel (DbPanel) it shows that error. see attached screeshots for more details..It seems that the bedezign\yii2\audit\panels\DbPanel::calculateTimings() is not adding the "traveHash" compared to the one from its parent yii\debug\panels\DbPanel::calculateTimings()
![Screenshot 2024-07-17 at 14-22-57 yii_base_ErrorException](https://github.com/user-attachments/assets/c6a14481-dd16-493c-ad4f-bb530c71e56d)

![Capture1234](https://github.com/user-attachments/assets/e5b5497a-0f0a-4470-85cd-886f2e4486da)
![Capture123](https://github.com/user-attachments/assets/409aa5b8-71b5-46c6-8dfe-e0462680c196)

**Comment by eluhr** [comment]
@khalid-eyez Could you please send me a section of your configuration in which you configure the audit module?

**Comment by khalid-eyez** [comment]
Here it is please !
![scshot](https://github.com/user-attachments/assets/501ff175-03fc-4abc-9bca-e23d1f6405e4)

**Comment by eluhr** [comment]
@khalid-eyez  It looks like the problem is not caused by the audit module but by the debug module. Have you possibly reconfigured some things in the debug module or changed files directly in the vendor?

**Comment by khalid-eyez** [comment]
@eluhr No...i did nothing on the debug module...i did not touch the vendor as well.

**Comment by schmunk42** [comment]
FYI: `traceHash` was added in 2.1.13 see https://github.com/yiisoft/yii2-debug/commit/5e4d49a2c5025cf88da8b3abde182d45ba6f4856#diff-cb48add4e37bc9b2ada1a193a9f3f0949db35f1cb2716d66f20410d9d1a49307R190

**Comment by khalid-eyez** [comment]
@schmunk42 as i can see there is also introduction of $queryDataProvider and $callerDataProvider. Look at lines 152 and 153. I also bumped onto that wall when i was trying to skip this current error.

**Comment by schmunk42** [comment]
PRs welcome, btw ;)

**Comment by khalid-eyez** [comment]
Any updates on this issue

**Comment by khalid-eyez** [comment]
Any updates at all?

**Comment by schmunk42** [comment]
Hi guys, sorry we just did not find any time until now.
We haven't forgotten it, just as a heads up.

**Comment by khalid-eyez** [comment]
Any updates on this issue... still not fixed even in the 1.2.8 (latest) version
